WalletWasabi.Tests/UnitTests/SmartLabelTests.cs

```diff
@@ -12,54 +12,88 @@ public void LabelParsingTests()
 	{
 		var label = new SmartLabel();
 		Assert.Equal("", label);
+		Assert.Equal(Array.Empty<string>(), label);
+		Assert.Equal(Array.Empty<string>(), label.AsSpan().ToArray());
 
 		label = new SmartLabel("");
 		Assert.Equal("", label);
+		Assert.Equal(Array.Empty<string>(), label);
+		Assert.Equal(Array.Empty<string>(), label.AsSpan().ToArray());
```

Comment:
It's intentional because an enumerator produced through `IEnumerable<T>.GetEnumerator()` and `ReadOnlySpan<T>.GetEnumerator()` are completely different. The last one implements no interfaces and cannot be allocated on the heap. Therefore, these two lines test different code paths.